### ISSUE TITLE
[Fix] data augmentation improves training/test score just for show

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#789]
## Summary
`_RegBase` (called by `Scenario.fit()` -> `RegressionHandler`) augments data as follows and this improves training/test score **just for show** (does not improve the accuracy of prediction in simulation/backtesting).
```Python
df = X_delayed.join(y, how="inner")
df = df.rolling(window=delay).mean().dropna().drop_duplicates()
```
This could be relaced with
```Python
df = X_delayed.join(y, how="inner").dropna().drop_duplicates()
```
because data augmentation makes the dataset biased.

## Codes

```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
oxcgrt_data = loader.oxcgrt()
# Scenario analysis
snl = cs.Scenario(country="Italy")
snl.register(jhu_data, extras=[oxcgrt_data])
snl.timepoints(today="05May2021")
snl.trend()
snl.estimate(cs.SIRF)
fit_df = snl.fit(name="Forecast")
print(fit_dict["score_train"])
print(fit_dict["score_test"])
snl.predict(name="Forecast")
snl.adjust_end()
snl.history("Confirmed")
snl.history("Recovered")
snl.history("Fatal")
```

## Outputs
train: 0.996189867761752
test: 0.988120415277019
![ita_12_history_Confirmed](https://user-images.githubusercontent.com/7270139/119364002-6b633e00-bce9-11eb-8338-ea58cab9fa0f.png)
![ita_14_history_Recovered](https://user-images.githubusercontent.com/7270139/119364006-6c946b00-bce9-11eb-98e2-53b5cd169308.png)
![ita_15_history_Fatal](https://user-images.githubusercontent.com/7270139/119364008-6d2d0180-bce9-11eb-83eb-df4582f0221c.png)

## Environment

- CovsirPhy version: 2.19.1-beta
- Python version: 3.9.5
- Installation: poetry
- System: WSL (Ubuntu)

**Comment by lisphilar** [comment]
With this change (#790),
score_train: 0.7975794166968244
score_test: 0.43356932618889077
![ita_12_history_Confirmed](https://user-images.githubusercontent.com/7270139/119364451-e298d200-bce9-11eb-8aac-c4116b7c2087.png)
![ita_14_history_Recovered](https://user-images.githubusercontent.com/7270139/119364460-e4fb2c00-bce9-11eb-9259-010621a2c008.png)
![ita_15_history_Fatal](https://user-images.githubusercontent.com/7270139/119364466-e6c4ef80-bce9-11eb-8506-0b1ca3232d9f.png)
